src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/packages.config

```diff
@@ -8,8 +8,8 @@
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.13" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.15" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.14" targetFramework="net452" />
```

Comment:
is this version bump intentional or accidental?

if the change are needed... I see this change taking place in a lot of places as part of this PR that shouldn't see it. Kindly explain why...